src/tests/amp/ampEpic.ts

```diff
@@ -1,65 +1,37 @@
 import { getLocalCurrencySymbol } from '../../lib/geolocation';
-import { AMPEpicResponse } from './ampEpicModels';
+import { AMPEpic } from './ampEpicModels';
 import { selectAmpEpic } from './ampEpicTests';
+import { AmpVariantAssignments } from '../../lib/ampVariantAssignments';
 
-async function ampDefaultEpic(geolocation?: string): Promise<AMPEpicResponse> {
-    const campaignCode = 'AMP_EPIC_AUGUST2020';
+async function ampFallbackEpic(geolocation?: string): Promise<AMPEpic> {
+    const campaignCode = 'AMP_FALLBACK_EPIC';
     const currencySymbol = getLocalCurrencySymbol(geolocation);
     return {
-        items: [
```

Comment:
if we deploy the server-side change first then presumably there will be a period where pages fail to handle the new payload? We can live with some downtime during the migration

Reply:
Yeah, we'll really need to deploy both simultaneously to minimise downtime